Source/Plugins/bsfOpenAudio/BsOAAudioSource.cpp

```diff
@@ -568,8 +582,10 @@ namespace bs
 
 			if (fillBuffer(mStreamBuffers[i], info, totalNumSamples))
 			{
-				for (auto& source : mSourceIDs)
+				for (auto& source : mSourceIDs) {
 					alSourceQueueBuffers(source, 1, &mStreamBuffers[i]);
+					alSourcePlay(source);
```

Comment:
This is the issue you mentioned, that we're forcing OpenAL to start playing again?

Reply:
Yes, ideally the source should not stop playing. In this case, I have to force the source to keep playing, but I am not yet sure why. As I mentioned before, I suspect it is due to the source running out of buffer contents too early because the audio thread doesn't keep up, but I don't know how to test this theory as of now...